packages/atomic/src/components/search/atomic-ipx-layout/ipx-layout.ts

```diff
@@ -0,0 +1,61 @@
+import {sectionSelector} from '../../common/atomic-layout-section/sections';
+
+const pagerSelector = 'atomic-pager';
+
+export function buildIPXLayout(element: HTMLElement) {
+  const id = element.id;
+  const layoutSelector = `atomic-ipx-layout#${id}`;
+
+  const interfaceStyle = `
+  ${layoutSelector} {
+    display: grid;
+    grid-template-rows: auto auto auto 8fr 1fr;
+    max-height: 100%;
+    box-sizing: border-box;
+    width: 500px;
+    height: 1000px;
+    margin-left: auto;
+    margin-right: auto;
+    box-shadow: 0px 3px 24px 0px #0000001a;
+  }`;
+
+  const search = `${sectionSelector('search')} {
```

Comment:
lol that's a way of doing it, I hope this value is converted to the related constant when the module is bundled 😅